cloudferry_devlab/cloudferry_devlab/bin/generate_load

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import ConfigParser
+import argparse
+import logging
+from logging.config import dictConfig
+
+import cloudferry_devlab.tests.config as conf
+from cloudferry_devlab.generate_load import Prerequisites
+from cloudferry_devlab.tests import base
+
+dictConfig(conf.logging_configuration)
+LOG = logging.getLogger(__name__)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(
+        description='Script to generate load for Openstack and delete '
+                    'generated objects')
+    parser.add_argument('--clean', help='clean objects, described'
+                                        ' in configuration.ini file',
+                        action='store_true')
+    parser.add_argument('--env', default='SRC',
+                        help='choose cloud: SRC or DST')
+    parser.add_argument('cloudsconf',
```

Comment:
add `type=argparse.FileType('r')`

and replace `confparser.read(_args.cloudsconf)` with `confparser.readfp(_args.cloudsconf)`